diff --git a/src/util.js b/src/util.js
index 308952d..456f35c 100644
--- a/src/util.js
+++ b/src/util.js
@@ -5,7 +5,10 @@ import { fileURLToPath } from 'node:url';
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 // explicit object instead of Object.fromEntries since the built-in type would loose the keys, better type: https://dev.to/svehla/typescript-object-fromentries-389c
-export const dataDir = s => path.resolve(__dirname, '..', 'data', s);
+export const dataDir = s => {
+  const basePath = process.env.DATA_DIR ?? path.resolve(__dirname, '..', 'data');
+  return path.resolve(basePath, s);
+};
 
 // modified path.resolve to return null if first argument is '0', used to disable screenshots
 export const resolve = (...a) => a.length && a[0] == '0' ? null : path.resolve(...a);
